src/cascade/input_data/db/bundle.py

```diff
@@ -13,6 +13,41 @@
 
 CODELOG, MATHLOG = getLoggers(__name__)
 
+BUNDLE_COLUMNS = [
```

Comment:
This method ensures the two are consistent. Again, would Pandas do this?

Reply:
Probably. We ended up here because I was just porting what the old importer does. I'm going to say we don't pursue that in this PR and do it in a more complete cleanup later. Mostly because I don't know panda's SQL layer well and I'd want to understand it better.

Reply:
Got it. OK. Visual inspection on those lists was good for me.